src/main/java/com/teamacronymcoders/epos/container/QuiverContainer.java

```diff
@@ -18,17 +18,18 @@
 
 public class QuiverContainer extends Container {
     @ObjectHolder("epos:quiver_container")
-    public static final ContainerType<QuiverContainer> TYPE = null;
-
+    public static ContainerType<QuiverContainer> TYPE;
+    private final PosInvHandler handler;
     private PlayerInventory player;
     private boolean hasPlayerInventory;
 
-    public QuiverContainer(int id, PlayerInventory playerInventory, PacketBuffer buffer) {
-        this(((QuiverItem) playerInventory.player.getHeldItemMainhand().getItem()).getHandler(playerInventory.player.getHeldItemMainhand()), playerInventory);
+    public QuiverContainer(int id, PlayerInventory player, PacketBuffer buffer) {
```

Comment:
PlayerInventory should be named that, if it's not actually PlayerEntity. Also you really should read from the buffer what hand the Quiver is in.

Reply:
How do I read hands from the buffer?
And how do I pass it in the actual thing?

Reply:
Pass it in as an int index would be the best.